iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/AbstractVoltageLevel.java

```diff
@@ -123,41 +124,45 @@ public VoltageLevel setHighVoltageLimit(double highVoltageLimit) {
     }
 
     @Override
-    public <T extends Connectable> FluentIterable<T> getConnectables(Class<T> clazz) {
+    public <T extends Connectable> Iterable<T> getConnectables(Class<T> clazz) {
```

Comment:
Why do you change FluentIterable into Iterable here?

Reply:
`FluentIterable` extends `Iterable` and is more powerful (can chain more easily like streams for example). However, when ensuring its elements are distinct by using `toSet()`, I am returning a `ImmutableSet` which extends `Iterable` too but not `FluentIterable`